tests/commonTemplates_test.go

```diff
@@ -431,7 +431,200 @@ var _ = Describe("Common templates", func() {
+							Verb:      "create",
+							Namespace: commonTemplates.GoldenImagesNSname,
+							Group:     "cdi.kubevirt.io",
+							Version:   "v1beta1",
+							Resource:  "datavolumes",
+						},
+					}),
+			)
+
+			table.DescribeTable("regular service account PVC RBAC", expectUserCan,
+				table.Entry("[test_id:4775]: ServiceAccounts with view role can view PVCs",
+					&authv1.SubjectAccessReviewSpec{
+						User:   regularSAFullName,
+						Groups: sasGroup,
+						ResourceAttributes: &authv1.ResourceAttributes{
+							Verb:      "get",
+							Namespace: commonTemplates.GoldenImagesNSname,
+							Resource:  "persistentvolumeclaims",
+						},
 					}))
+			table.DescribeTable("regular service account RBAC", expectUserCannot,
+				table.Entry("[test_id:4776]: ServiceAccounts with only view role cannot create PVCs",
+					&authv1.SubjectAccessReviewSpec{
+						User:   regularSAFullName,
+						Groups: sasGroup,
+						ResourceAttributes: &authv1.ResourceAttributes{
+							Verb:      "create",
+							Namespace: commonTemplates.GoldenImagesNSname,
+							Resource:  "persistentvolumeclaims",
+						},
+					}),
+				table.Entry("[test_id:4846]: ServiceAccounts with only view role cannot delete PVCs",
+					&authv1.SubjectAccessReviewSpec{
+						User:   regularSAFullName,
+						Groups: sasGroup,
+						ResourceAttributes: &authv1.ResourceAttributes{
+							Verb:      "delete",
+							Namespace: commonTemplates.GoldenImagesNSname,
+							Resource:  "persistentvolumeclaims",
+						},
+					}),
+				table.Entry("[test_id:4879]: ServiceAccounts with only view role cannot create any other resources other than the ones listed in the View role",
+					&authv1.SubjectAccessReviewSpec{
+						User:   regularSAFullName,
+						Groups: sasGroup,
+						ResourceAttributes: &authv1.ResourceAttributes{
+							Verb:      "create",
+							Namespace: commonTemplates.GoldenImagesNSname,
+							Resource:  "pods",
+						},
+					}),
+			)
+			Context("With Edit permission", func() {
+				BeforeEach(func() {
+					editObj := &rbac.RoleBinding{
+						ObjectMeta: metav1.ObjectMeta{
+							Name:      "testeditimage",
+							Namespace: commonTemplates.GoldenImagesNSname,
+						},
+						Subjects: []rbac.Subject{
+							{
+								Kind:      "ServiceAccount",
+								Name:      regularSA.GetName(),
+								Namespace: strategy.GetNamespace(),
+							},
+						},
+						RoleRef: rbac.RoleRef{
+							Kind:     "ClusterRole",
+							Name:     commonTemplates.EditClusterRoleName,
+							APIGroup: "rbac.authorization.k8s.io",
+						},
+					}
+					Expect(apiClient.Create(ctx, editObj)).ToNot(HaveOccurred(), "Failed to create RoleBinding")
+				})
+				table.DescribeTable("should verify resource permissions", expectUserCan,
+					table.Entry("[test_id:4774]: ServiceAcounts with edit role can create PVCs",
+						&authv1.SubjectAccessReviewSpec{
+							User:   regularSAFullName,
+							Groups: sasGroup,
```

Comment:
I think CI fails because you specify `Groups`.
According to the documentation, this SAR will check if `regularSA` received the appropriate `create` permission from the `system:serviceaccounts` group, which is not the case because there is no edit permissions for `system:serviceaccounts`, and the `RoleBinding` you created refers to `regularSA`. So `Groups` should be empty:
```
	// User is the user you're testing for.
	// If you specify "User" but not "Groups", then is it interpreted as "What if User were not a member of any groups
	// +optional
	User string `json:"user,omitempty" protobuf:"bytes,3,opt,name=user"`
	// Groups is the groups you're testing for.
	// +optional
	Groups []string `json:"groups,omitempty" protobuf:"bytes,4,rep,name=groups"`
	// Extra corresponds to the user.Info.GetExtra() method from the authenticator.  Since that is input to the authorizer
	// it needs a reflection here.
```
Also, maybe change `regularSA` to `privilegedSA` in this context for readability